plugins/services/src/js/containers/services/ServicesTable.js

```diff
@@ -266,13 +266,14 @@ class ServicesTable extends React.Component {
     const serviceStatus = service.getStatus();
     const serviceStatusClassSet = StatusMapping[serviceStatus] || "";
     const tasksSummary = service.getTasksSummary();
-    const { tasksRunning } = tasksSummary;
-
+    const tasksRunning = service.getTaskCount();
     const isDeploying = serviceStatus === "Deploying";
 
-    const conciseOverview = ` (${tasksRunning}/${instancesCount})`;
+    let conciseOverview = ` (${tasksRunning})`;
     let verboseOverview = ` (${tasksRunning} ${StringUtil.pluralize("Instance", tasksRunning)})`;
+
     if (tasksRunning !== instancesCount) {
+      conciseOverview = ` (${tasksRunning}/${instancesCount})`;
```

Comment:
The code was like this already here but would be nice to refactor to ternary too.
```
const conciseOverview = tasksRunning !== instancesCount
      ? ` (${tasksRunning}/${instancesCount})`
      : ` (${tasksRunning})`;
    const verboseOverview = tasksRunning !== instancesCount
      ? ` (${tasksRunning} of ${instancesCount} Instances)`
      : ` (${tasksRunning} ${StringUtil.pluralize("Instance", tasksRunning)})`;
```